interfaces/nginx_route/v0/schemas/requirer.json

```diff
@@ -0,0 +1,176 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "$id": "https://canonical.github.io/charm-relation-interfaces/interfaces/nginx_route/schemas/requirer.json",
+    "type": "object",
+    "title": "`nginx-route` requirer schema",
+    "description": "The `nginx-route` root schema comprises the entire requirer databag for this interface.",
+    "default": {},
```

Comment:
Is this right? Passing an empty value is a valid value?

Reply:
There's no default value in this relation.

Reply:
I removed this `default` too.